### DISCUSSION TITLE
CUDA memory problems - connected to amount of embeddings, LORAs etc?

### DISCUSSION THREAD

**Discussion by rodwd** [TheLastBen/fast-stable-diffusion#2352] [Q&A]
I'm wondering. Do recently get more and more CUDA memory errors once I enable Hi.Res upscaling. Could that be connected to the amount of embeddings (300+), LORAs etc?
Or do I need to move to the higher end GPUs with Colab?

**Comment by TheLastBen** [comment]
try again with the latest notebook

**Comment by rodwd** [comment]
Works again now, thanks.

**Reply by rodwd** [reply]
Well - again not working. Don't know why but after a while I always get those memory errors if I try to use Hi.Res. Although I've changed the GPU to V100 which ups the system RAM to 25 GB. GPU RAM stays at 16 GB.

**Reply by TheLastBen** [reply]
you can change the RAM without changing the GPU
